src/pages/en/guides/content-collections.md

```diff
@@ -32,15 +33,52 @@ export default defineConfig({
 });
 ```
 
-You will also need to update `tsconfig.json`. Add `"strictNullChecks": true` under `compilerOptions`.
+### Update TypeScript configuration
 
-```json title="tsconfig.json" ins={3}
-{
-  "compilerOptions": {
-    "strictNullChecks": true
+To benefit from the full TypeScript and autocompletion features of content collections, you may also need to update `tsconfig.json`. Proper TypeScript configuration is required to [use schemas with your collections](#defining-a-collection-schema).
+
+To enable the setting necessary for schemas, follow the instructions below based on your Astro project's current [TypeScript configuration](/en/guides/typescript/#setup). 
+
+<TypeScriptSettingTabs>
+  <Fragment slot="relaxed">
+  If you are using Astro's `base` setting (e.g. you set up your project with "relaxed" TypeScript), add `"strictNullChecks": true` under `compilerOptions`. 
```

Comment:
Small tweaks.

- Say “extending” `base` to reflect it’s literally in the `extends` option
- Tweaked the parenthetical to be clear “relaxed” refers to the `create astro` terminology

```suggestion
  If you are extending Astro's `base` TypeScript config (e.g. you chose "relaxed" TypeScript when creating your project with `create astro`), add `"strictNullChecks": true` under `compilerOptions`.
```